### ISSUE TITLE
Contributions guidance

### ISSUE THREAD

**Issue by mumby0168** [IEvangelist/azure-cosmos-dotnet-repository#368]
It would be nice to provide a bit more information for people wanting to contribute to the library around how to get started and maybe we can do something more to explain the acceptance tests and how they work. The configuration is not obvious.

It would be nice to add additional bits to contribution guidelines see this as an example:

https://github.com/fluentassertions/fluentassertions/blob/develop/CONTRIBUTING.md

**Comment by IEvangelist** [comment]
Yeah, I love that idea! And it seems like the FluentAssertions has a nice example, thanks for driving this.
